webapp-django/crashstats/crashstats/models.py

```diff
@@ -799,7 +798,6 @@ class RawCrash(SocorroMiddleware):
         "MacMemoryPressureNormalTime",
         "MacMemoryPressureSysctl",
         "MacMemoryPressureWarningTime",
-        "MinidumpSha256Hash",
```

Comment:
This was a "fake annotation". The processor now pulls this from the `dump_checksums`.